src/test/java/com/google/jenkins/plugins/k8sengine/KubectlWrapperVerboseLoggingTest.java

```diff
@@ -1,92 +0,0 @@
-/*
- * Copyright 2019 Google LLC
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.google.jenkins.plugins.k8sengine;
-
-import com.google.common.collect.ImmutableList;
-import hudson.FilePath;
-import hudson.Launcher;
-import hudson.Launcher.ProcStarter;
-import java.io.IOException;
-import java.io.OutputStream;
-import org.junit.Test;
-import org.junit.runner.RunWith;
-import org.mockito.Mockito;
-import org.mockito.junit.MockitoJUnitRunner;
-
-/** Tests for verifying the use of the verboseLogging flag in {@link KubectlWrapper}. */
-@RunWith(MockitoJUnitRunner.class)
-public class KubectlWrapperVerboseLoggingTest {
```

Comment:
I'd prefer disabling and tweaking this test later rather than deleting it

Reply:
I don't have strong objections to disabling it. However, I'm not sure this test has value as is and might not be worth the effort it would take to get it to work. It doesn't actually make assertions and thus provides little value IMHO. If it were re-factored to to make actual assertions I could see an argument for keeping. Let me know what you think.

Reply:
I think I faced problems trying to make this more specific. It tests behavior rather than state, with the one implicit assertion being the expected IOException. Until there's a better way to test this it's fine to remove.